downloads/platform.md

```diff
@@ -23,6 +23,13 @@ On Mac, a `Julia-<version>.dmg` file is provided, which contains Julia.app. Inst
 
 Uninstall Julia by deleting Julia.app and the packages directory in ~/.julia. Multiple Julia.app binaries can co-exist without interfering with each other. If you would also like to remove your preferences files, remove `~/.juliarc.jl`.
 
+An alternative to installing Julia from the downloaded .dmg is by using the [Homebrew package manager](https://brew.sh/). 
+In your terminal, run
+
+    brew cask install julia
+
+You can also install from the [`staticfloat/julia` tap](https://github.com/staticfloat/homebrew-julia). Note that this tap is still slightly experimental and involves more compiling.
```

Comment:
It shouldn't actually require any compiling though, right? I thought @staticfloat built binaries for his Julia formula regularly so that users wouldn't have to compile anything.